app/models/comment.rb

```diff
@@ -1,4 +1,8 @@
 class Comment < ApplicationRecord
   belongs_to :author, foreign_key: :author_id, class_name: 'User'
   belongs_to :post, class_name: 'Post'
+
+  def update_post_comment_counter
```

Comment:
[OPTIONAL]
- Since you don't need to access this method outside the class, you can make it private
```ruby
  # app/models/post.rb
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```
The same concept can be applied to the `Comment` and `Post` class methods.